doc/nrf/ug_radio_coex.rst

```diff
@@ -0,0 +1,196 @@
+.. _ug_radio_coex:
+
+Working with WiFi Coexistence in Short Range Radio applications
+###############################################################
+
+.. contents::
+   :local:
+   :depth: 4
+
+Short Range RF technologies such as Bluetooth LE or 802.15.4 (from now on refered to as SR) use different radios than WiFi. However, if both SR and WiFi
+attempt to transmit at the same time RF waves will interfere with each other leading to decreased performance and unneeded higher power
+consumption. Also, in some cases (such as receiving ACK) no radio should be transmitting in order to ensure correct reception.
+
+In order to mitigate coexistence issues and improve coexistence performance a Packet Traffic Arbiter (PTA) is used. Both SR and WiFi request access to RF
+and PTA grants/denies that access.
+
+This guide describes how to add WiFi Coexistence (CX) support to your SR application in |NCS|. In order to do it, both
+:ref:`Software support` and :ref:`Hardware description` must be added.
+
+Software support
+****************
+
+If your application uses SR protocols and requires coexistence with WiFi, you must enable :ref:`nrfxlib:mpsl_cx` in the
+:ref:`nrfxlib:mpsl` (MPSL) library.
+
+CX with MPSL
+============
+
+The MPSL library provides an implementation for the WiFi Coexistence for SR applications.
+To use this implementation, your application must use a protocol driver that enables the WiFi Coexistence (CX) feature.
+The library provides multiprotocol support, but you can also use it in applications that require only one protocol.
+To avoid conflicts, check the protocol documentation to see if the protocol uses CX support provided by MPSL.
+
+.. note::
+   Currently, the following protocols use CX support provided by MPSL:
+
+   * :ref:`ug_thread`
+   * :ref:`ug_zigbee`
+
+.. _ug_radio_cx_requirements:
+
+Enabling MPSL
+-------------
+
+Before you add the devicetree node in your application, complete the following steps:
+
+   The MPSL library provides API to handle signals coming from PTA and controlling requests send to PTA.
+   See :ref:`nrfxlib:mpsl_lib` in the nrfxlib documentation for details.
+
+1. Add support for the MPSL library in your application.
```

Comment:
Actually, this holds only for single-core devices such as the nRF52 SoCs. On nRF53, the application doesn't need to support the MPSL library - only the network core uses it. `CONFIG_MPSL` should not be set on the application core.